tests/spread/testcraft/init-pack/task.yaml

```diff
@@ -12,5 +12,15 @@ execute: |
   testcraft pack
   testcraft clean
 
+  if [[ $(find . -maxdepth 1 -name '*-default.partitioncraft') == "" ]]; then
+    echo "No default partition output" >> /dev/stderr
+    exit 1
+  fi
+
+  if ! [[ $(find . -maxdepth 1 -name '*-mushroom.partitioncraft') == "" ]]; then
+    echo "No mushroom partition output" >> /dev/stderr
+    exit 1
+  fi
+
```

Comment:
I think this change should be in `tests/spread/partitioncraft/init-pack/task.yaml`.